stream_alert/shared/lookup_tables/errors.py

```diff
@@ -0,0 +1,15 @@
+
+class LookupTablesError(RuntimeError):
+    pass
+
+
+class LookupTablesInitializationError(LookupTablesError):
+    pass
+
+
+class LookupTablesCommitError(LookupTablesError):
```

Comment:
some doc strings on what these different errors are for might be nice. up to you